registry/index/templates/index.html

```diff
@@ -16,11 +16,16 @@ <h1 class="display-r">Open Science Pool Token Service</h1>
                     <p class="lead">
                         The Open Science Pool token service provides system administrators
                         with the ability to request and receive tokens necessary to authenticate
-                        worker node containers with the Open Science Pool.
+                        Access Points and worker node containers with the Open Science Pool.
```

Comment:
It seems weird to have access points capitalized but worker node containers uncapitalized. You should probably uncapitalize access points -- the capitalized version makes me want to add a (TM) afterward

Reply:
Good call but I think only `Worker Node` should be capitalized and I switched it to Execution Point to align with the PATh glossary